### PR TITLE
Add Webpack to automatically watch and build changes.

### PR DESCRIPTION
**Pull request by chronicIntrovert**
- [ ] I have put a reference to a related issue in the description
- [x] I have created the pull request from the correct version branch
- [x] I have ensured the branch is rebased and/or there are no merge conflicts

@jyntran - used chrome-extension-cli to add Webpack configuration for automatically building and watching changes. This makes development much easier than having to manually manage loading unpacked version after update. Let me know what you think.

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const merge = require('webpack-merge');
+
+const common = require('./webpack.common.js');
+const PATHS = require('./paths');
+
+// Merge webpack configuration files
+const config = merge(common, {
+  entry: {
+    popup: PATHS.src + '/popup.js',
+    contentScript: PATHS.src + '/contentScript.js',
```
Comment (jyntran) [inline config/webpack.config.js]: I found this only works for me if both `contentScript` are changed to `contentscript`. Can you confirm this?

File: README.md [hunk 2]
```diff
@@ -33,8 +34,10 @@ Feel free to contribute or post issues to the [GitHub repository](https://github
 
 ## Note for Developers
 
-- You will need a Last.fm API key and secret to run it: http://www.last.fm/api
-- Place your key and secret in `api/config.js`. The file `config.sample.js` is provided as a template.
+-   You will need a Last.fm API key and secret to run it: http://www.last.fm/api
+-   Place your key and secret in `api/config.js`. The file `config.sample.js` is provided as a template.
+-   To get started, install Node ^10.16.3 with npm ^6.12.0, then run `npm i && npm watch`
```
Comment (jyntran) [inline README.md]: Almost works but I needed to run npm run watch instead. Does this also work for you?

**Comment by jyntran** [comment]
@imthinhvu This is a great idea, thanks for thinking of this! I have a few suggestions, but otherwise it's working.

File: config/webpack.config.js (hunk 1 above)
Comment (imthinhvu) [reply config/webpack.config.js]: I changed the spelling. I hope that's ok!

File: README.md (hunk 2 above)
Comment (imthinhvu) [reply README.md]: Good catch on the typo. I corrected it.

File: config/webpack.config.js (hunk 1 above)
Comment (jyntran) [reply config/webpack.config.js:12]: That should be fine as it's now in proper camelcase. What's left is to rename the file `src/contentscript.js` to match. Thanks!

**Review by jyntran — APPROVED** [review]

**Pull request merged**